python/test/t_Function_linearity.py

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env python
```

Comment:
Please create a Notebook example from this unit test. This will explain how linear functions can be used in the Bonmin context. Otherwise, this feature will be unknown to users.